initfiles/componentfiles/configschema/xsd/sparkthor.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+        xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+        xmlns:hpcc="someuri">
+    <xs:include schemaLocation="types.xsd"/>
+    <hpcc:insert hpcc:schemaPath="/Environment/Software">
+        <xs:element name="SparkThorProcess" hpcc:itemType="sparkthor" hpcc:displayName="Spark Thor" maxOccurs="1" hpcc:class="component"
+                    hpcc:docid="Th.t1">
+            <xs:complexType>
+
+                <xs:sequence>
+                    <xs:element name="Instance" maxOccurs="unbounded" hpcc:class="elementSet"
+                                hpcc:displayName="Bound Computers">
+                        <xs:complexType>
+                            <xs:attribute name="computer" type="xs:string" hpcc:displayName="Computer Type" hpcc:sourceKey="computertype_name"
+                                          use="required" hpcc:tooltip="Computer type"/>
+                            <xs:attribute name="netAddress" hpcc:displayName="Net Address" type="ipV4Address" use="required"
+                                          hpcc:readOnly="true"
+                                          hpcc:sourceKey="computer_netaddress"
+                                          hpcc:mirrorFrom="/Environment/Hardware/Computer[@netAddress]"/>
+                            <xs:attribute name="directory" type="xs:string"
+                                          hpcc:presetValue="c$\dafilesrv" hpcc:hidden="true"/>
+                        </xs:complexType>
+                    </xs:element>
+                </xs:sequence>
+
+                <xs:attributeGroup ref="buildInfo"/>
+                <xs:attribute name="name" type="xs:string" use="required" hpcc:tooltip="Name for this process"/>
+                <xs:attribute name="ThorClusterName" type="xs:string" hpcc:displayName="Thor Cluster Name" use="required"
```

Comment:
toolitp: "Thor cluster for spark workers to attach to"

Reply:
Updated, New tip may be a bit pedantic since I removed the ending preposition.